backend/src/controllers/pendingCourses/approvePendingCourse.ts

```diff
@@ -45,8 +47,11 @@ const approvePendingCourse = async (
       await arrangeAssessment(clientCourseId);
     }
 
-    results.updateStatus = `Course was approved ${withAssessment ? 'with' : 'without'} assessment.`;
     next();
+
+    res.json({
+      updateStatus: `Course was approved ${withAssessment ? 'with' : 'without'} assessment.`,
```

Comment:
as you wrote  we don't need "update status", just plane string in res

Reply:
done